registration-processor/post-processor/registration-processor-printing-stage/src/main/java/io/mosip/registration/processor/print/stage/PrintingStage.java

```diff
@@ -354,6 +353,9 @@ private CredentialRequestDto getCredentialRequestDto(String regId) {
 		credentialRequestDto.setIssuer(env.getProperty("mosip.registration.processor.issuer"));
 
 		credentialRequestDto.setEncryptionKey(generatePin());
+		additionalAttributes.put("credential_id",regId);
+		additionalAttributes.put("template",env.getProperty("mosip.registration.processor.uincard.template"));
```

Comment:
This property will give back the template name or full template itself ?  I don't see this property in config PR

Reply:
also handle the null check or get it as @value with default value as something

Reply:
taken care